### PR TITLE
Add v1.3.1 release note

### PR DESCRIPTION
**Pull request by bk201**

### CODE REVIEW

File: v1.3.1.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+Harvester can be installed using the [ISO image](https://docs.harvesterhci.io/v1.3/install/index), a [bootable USB drive](https://docs.harvesterhci.io/v1.3/install/usb-install), and [PXE boot](https://docs.harvesterhci.io/v1.3/install/pxe-boot-install). For more information, see the [Installation](https://docs.harvesterhci.io/v1.3/install/requirements) section of the documentation.
+
+
+## Upgrade
+
+Harvester only allows upgrades from supported versions. For more information about upgrade paths and procedures, see [Upgrading Harvester](https://docs.harvesterhci.io/v1.3/upgrade/index).
+
+> [!NOTE]
+> Harvester support upgrading from v1.2.2 or v1.3.0 to v1.3.1.
```
[the remark this answers is not in the document]
Comment (jillian-maroket) [reply v1.3.1.md:59]: The sentence that I added is similar to the one from the v1.2.2 release notes. If you think it's unnecessary, feel free to remove it.

**Review by jillian-maroket — COMMENTED** [review]
I have a small suggestion for the upgrade note.

File: v1.3.1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bk201) [reply v1.3.1.md:59]: The change looks much more professional, thanks.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Review by innobead — COMMENTED** [review]
LGTM

**Pull request merged**